### DISCUSSION TITLE
cut_off voltage is ignored in experiment

### DISCUSSION THREAD

**Discussion by masoodtamaddon** [pybamm-team/PyBaMM#1523] [Q&A] [answered]
Hi

I have tested two different scenarios: one discharge using **Experiment** and another discharge using the imbedded current profile (e.g. for Yang2017 DFN model) and noticed that when I use Experiment, the cut-off voltage is ignored in the solution. Is there anyway to work around this issue? 

BR,
/Masood

See the following script: 
--------------------------

import pybamm as pb
import numpy as np

model = pb.lithium_ion.Yang2017()
param = model.default_parameter_values
param.update({"Lower voltage cut-off [V]": 3.5} , check_already_exists=True)
output_variables = ["Current [A]", "Terminal voltage [V]"]

# test1: discharge using experiment
exp = pb.Experiment(
    [
        "Discharge at C/1 for 1 hours"
    ]
)

sim_exp = pb.Simulation(model, 
parameter_values=param, 
experiment = exp, 
solver=pb.CasadiSolver(mode="safe")
)
solution_exp = sim_exp.solve()
sim_exp.plot(output_variables=output_variables)

# test2: discharge using param
t_eval = np.linspace(0, 3600, 100)
sim = pb.Simulation(model, 
parameter_values=param, 
solver=pb.CasadiSolver(mode="safe")
)
solution = sim.solve(t_eval)
sim.plot(output_variables=output_variables)

**Comment by valentinsulzer** [comment, marked as answer]
To specify a cut-off voltage in the experiment you have to do "Discharge at C/1 for 1 hours or until 3.5V" or "Discharge at C/1 until 3.5V"
